### ISSUE TITLE
ERROR: '>=' not supported between instances of 'NoneType' and 'int'

### ISSUE THREAD

**Issue by quicktrick** [fail2ban/fail2ban#2432]
fail2ban 0.11 (latest github), Python 3.6, FreeBSD.

There are some error records in the log like this:
```
2019-05-27 19:04:08,371 fail2ban.observer       [22526]: ERROR   '>=' not supported between instances of 'NoneType' and 'int'
```
They usually occur after the records like this:
```
2019-05-26 00:27:56,013 fail2ban.observer       [22526]: INFO    [postfix] Found 185.222.211.10, bad - 2019-05-26 00:27:55, 2 # -> 3.0, Ban
2019-05-26 00:27:56,017 fail2ban.observer       [22526]: ERROR   '>=' not supported between instances of 'NoneType' and 'int'
.....
2019-05-27 17:15:59,970 fail2ban.observer       [22526]: INFO    [postfix] Found 89.248.167.131, bad - 2019-05-27 17:15:59, 1 # -> 2.0
2019-05-27 17:15:59,977 fail2ban.observer       [22526]: ERROR   '>=' not supported between instances of 'NoneType' and 'int'
.....
2019-05-27 19:04:08,371 fail2ban.observer       [22526]: INFO    [postfix] Found 37.49.227.129, bad - 2019-05-27 19:04:07, 1 # -> 2.0
2019-05-27 19:04:08,371 fail2ban.observer       [22526]: ERROR   '>=' not supported between instances of 'NoneType' and 'int'
```
But not all the "bad" records have such errors after them. 

I've found a similar issue [here](https://sourceforge.net/p/fail2ban/mailman/message/36624352/), but there was no reply to it. I've also found some [explanation](https://stackoverflow.com/questions/50139140/how-do-i-resolve-not-supported-between-instances-of-int-and-nonetype-i), why that may happen.

**Comment by sebres** [comment]
Ok, this is always nice to listen that every next python version becomes fewer backwards compatibility as previous.
*(Before 3.x `None >= 1` would simply return False).*

Must check where it could be.

**Comment by sebres** [comment]
Should be fixed now in 7c7dfab8dbd612615c9a532f3cf3df514654d041.
Please update and confirm it'd help. 
Thanks!

**Comment by quicktrick** [comment]
@sebres  Everything seems to be good now. Thanks a lot, Sergey!
